tests/test_placeholder_validator.py

```diff
@@ -189,3 +189,24 @@ def test_validation_answer_and_transform_unit_match(
         arguments=arguments, transform_type=transform_type
     )
     assert validator.errors == expected_error
+
+
+@pytest.mark.parametrize(
```

Comment:
since there is only one test case - I don't think theres any need to parametrize this, the expected errors can just be in the function

Reply:
I thinking here about the future and doing it with parameters if we want to expand, similar to the one above but ok will change it now.

Reply:
sorry, now that you've added more tests it does make sense 😁